app/models/concerns/content_matching_operations.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ContentMatchingOperations
```

Comment:
Imho mixiny davaju zmysel ked to pojde do viacerych tried, toto mi pride, ze cela tato logika by mala byt v action samotnej.

Reply:
Zaviedol som to prave preto, ze to ide do dvoch tried - message_object a nested_message_object. Mam to teda stahovat do action?

Reply:
No praveze mne pride ze by to malo byt len v action a prinajlepsom message_object alebo nested_message_object budu mat nejaku malu metodu ktora bude vytahovat obsah pre toto. nedava mi zmysel uplne rovnake metody tlacit do dvoch tried, to smrdi.